v1/messages/src/requestHandlers.ts

```diff
@@ -107,25 +107,20 @@ function parseSlugFromURI(uri: string, resource: string): string {
   return "";
 }
 
-function parseAuthorizationHeader(auth: string): string | null {
-  if (auth === undefined) {
-    return null;
-  }
+function parseBasicAuth(auth: string): string | null {
```

Comment:
nit: I would name this `parseUserId` or something, because that's more accurate for what you are actually parsing